src/components/pages/Desk/__specs__/Desk.spec.jsx

```diff
@@ -270,6 +270,29 @@ describe('src | components | Desk', () => {
       const responseFromApi = await screen.findByText('error message')
       expect(responseFromApi).toBeInTheDocument()
     })
+
+    it('should display an error message when the booking invalidation has failed', async () => {
+      // given
+      jest.spyOn(props, 'getBooking').mockRejectedValue({
+        json: jest.fn(() => Promise.resolve({ booking: 'token is already validated' })),
```

Comment:
```suggestion
        json: () => Promise.resolve({ booking: 'token is already validated' }),
```

Jest n'est pas utile dans cas si je ne m'abuse (tu ne mockes pas ni ne stub)